rustls/Cargo.toml

```diff
@@ -23,7 +23,7 @@ log = { version = "0.4.4", optional = true }
 once_cell = { version = "1.16", default-features = false, features = ["alloc", "race"] }
 ring = { version = "0.17", optional = true }
 subtle = { version = "2.5.0", default-features = false }
-webpki = { package = "rustls-webpki", version = "0.102.2", features = ["alloc"], default-features = false }
+webpki = { package = "rustls-webpki", version = "0.102.4", features = ["alloc"], default-features = false }
```

Comment:
I think to fix the ["Check minimum versions of direct dependencies"](https://github.com/rustls/rustls/actions/runs/9117850386/job/25070188927) CI failure you'll need a commit like https://github.com/cpu/rustls/commit/0eb3df6a37472559d74314143c08530ae40eebf1 to update the pki-types version :thinking: